common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -177,7 +178,7 @@ public Object tryParseStringValue(String value) {
         try {
             Class<?> wrapped = ClassUtils.primitiveToWrapper(((Class<?>) fieldType));
             if (wrapped.isEnum()) {
-                return Enum.valueOf((Class<? extends Enum>) wrapped, value);
+                return EnumUtils.getEnumIgnoreCase((Class<? extends Enum>) wrapped, value);
```

Comment:
Kind of unrelated to the PR? Does this improve anything? `/config set`?

Reply:
Yes, otherwise `/config set TelemetryFeature crashReport true` did not work, but had to be `/config set TelemetryFeature crashReport TRUE`, once I made an enum out of it.

Reply:
So while it is sort of unrelated, it is necessary for this PR.